### DISCUSSION TITLE
Minimum requirements?

### DISCUSSION THREAD

**Discussion by seeergiofdez** [valinet/ExplorerPatcher#1543] [Q&A]
Hi, I'm trying to use this program in Windows build 14393 (Windows 10 1607), but I've got an error opening the explorer patches from the start menu. Is it related to my windows version? I installed successfully on build 17763

**Comment by pyrates999** [comment]
You are running a version of windows 10 from 2017 that is no longer supported.

Please do the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. update to windows 10 22h2.  You can do this from settings -> updates & security

**Reply by seeergiofdez** [reply]
Yeah I know, I use 14393 (2016 LTSB) on purpose... So my question was if there is minimum requirements for ExplorerPatcher to work? I see that it seems so... But thanks!
